src/simulation/navigation/simpleNav/_UnitTest/test_simpleNav.py

```diff
@@ -160,6 +171,52 @@ def simple_nav_sim(show_plots, noise=False, number_steps=1, sigma_test=3):
     ephemeris_attitude_output = ephemeris_data_log.sigma_BN
     ephemeris_rate_output = ephemeris_data_log.omega_BN_B
 
+    test_packets = accel_data_log.accPkts
+    rate_array = np.zeros([number_steps*simple_nav_object.numberOfGyroBuffers, 3])
+    accel_array = np.zeros([number_steps*simple_nav_object.numberOfGyroBuffers, 3])
+    time_list = []
+    for i in range(number_steps):
+        for j in range(simple_nav_object.numberOfGyroBuffers):
+            time_list.append(test_packets[i]["accPkts[" + str(j) + "].measTime"] - test_packets[i]["accPkts[0].measTime"])
+            rate_array[i*simple_nav_object.numberOfGyroBuffers+j, :] = np.array(test_packets[i]["accPkts[" + str(j) + "].gyro_B"])
+            accel_array[i*simple_nav_object.numberOfGyroBuffers+j, :] = np.array(test_packets[i]["accPkts[" + str(j) + "].accel_B"])
+    avg_rate = np.mean(rate_array, axis=0)
+    avg_accel = np.mean(accel_array, axis=0)
+    avg_time = np.mean(time_list)
+    std_rate = np.std(rate_array, axis=0)
+    std_accel = np.std(accel_array, axis=0)
+
+    np.testing.assert_allclose(avg_time,
+                               (1E9/simple_nav_object.gyroFrequencyPerSecond)*(simple_nav_object.numberOfGyroBuffers - 1)/2,
+                               atol=3*simple_nav_object.gyroStandardDeviation/np.sqrt(number_steps),
+                               equal_nan=False,
+                               err_msg="Middle of the time measurement failed",
```

Comment:
:large_orange_diamond: Let's edit these error messages to be more descriptive of what was expected. We can leave out any mention of failure as that is taken care of by the assertion apparatus.

Reply:
done